docs/DeveloperGuide.md

```diff
@@ -6,7 +6,152 @@
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+Here is a quick overview of the design of the application:
+
+![](diagrams/class/all.png)
```

Comment:
At first glance, the diagram for the entire design of the application is less readable as the font size is too small. Might want to enlarge the diagram as a whole if all information needs to be kept.
![image](https://user-images.githubusercontent.com/65325165/160956858-2d3c0203-5cac-4241-a04a-b8ac5f9639a6.png)